### DISCUSSION TITLE
does slint good for realtime data visualization ? (show in web browser)

### DISCUSSION THREAD

**Discussion by anyfig** [slint-ui/slint#2801] [Q&A] [answered]
hey guys ! 
the slint WebAssembly demos do really amaze me , with high performance and the same-as-desktop gui.
sad to see `the web platform is not one of our primary target platforms` in readme.md. 
does slint good for realtime data visualization ? (2d/3d chart and Video Streaming show in web browser)
or , is there any other framework better ?

**Comment by ogoffart** [comment, marked as answer]
Yes, the Web platform is maintained so we can have cool demos, but not what we recommend in production.
The reason is that we draw everything on a `<canvas>` so this doesn't use the browser text / accessibility, and other things we get if we were using the DOM.
If that's working for your use-case, then you can use Slint.
